class ValueError(Exception) :
    pass

class RepoError(Exception) :
    pass

class RepoErrorComputer(Exception) :
    pass